### ISSUE TITLE
Please help test release candidate 2.0.0

### ISSUE THREAD

**Issue by lstein** [invoke-ai/InvokeAI#930]
As we move towards a release of InvokeAI version 2.0, you can help by running a series of preflight checks against the relase candidate on your system. 

Instructions can be found at #914 . Please use the comment feature to enter your feedback. Good luck, and happy testing!

**Comment by Domino9752** [comment]
Say Hey-   
I tried and failed to preview version 2.0, and got distracted. Now I had the same issue with the release, so failure was not an option.  The problem was trivial, just a need to manually install Eventlet, flask and flask_socketio (Win10, anaconda [and yes, I had to delete he contents of /src  too] ) . My comment is that I still fail to see where I should drop a simple note for such a small "spit and polish" observation. When I see yet another obsolete reference to the "stable-diffusion" directory in one of the MD files, how does one best informally "drop-a-line" about it?
